content/en/docs/contribute/new-content/new-features.md

```diff
@@ -95,14 +95,14 @@ deadlines.
 
 ### Open a placeholder PR
 
-1. Open a pull request against the
+1. Open a **draft** pull request against the
```

Comment:
"draft" here looks like a relative term to "ready for review" below, if I'm understanding this correctly.

Please be specific how to mark a PR as a "draft". Do you mean prefixing the PR title with "WIP -" or something?

Reply:
It's a stage of a pull request that is offered by Github. I can see that it may be not clear, so I added steps on how to mark a pull request as "draft" and "ready for review" in the corresponding sections. Could you take a look again? Does this make more sense?